src/utils/gauges.ts

```diff
@@ -154,11 +156,13 @@ export async function getGaugeData(
         gaugeContract.working_supply(),
       ),
     )
-    const gaugeWorkingBalancesPromise = ethCallProvider.tryAll(
-      gaugeMulticallContracts.map((gaugeContract) =>
-        gaugeContract.working_balances(account),
-      ),
-    )
+    const gaugeWorkingBalancesPromise = account
```

Comment:
To make this shorter you can also do

```
    const gaugeWorkingBalancesPromise = account && ethCallProvider.tryAll(
          gaugeMulticallContracts.map((gaugeContract) =>
            gaugeContract.working_balances(account),
          ),
        )
```

Reply:
Because `tryAll` 's return value is `Promise<(O | null)[]>`, so if the promise is not exist, then we need to give `null` value, but && will make `undefined`.

Reply:
Good point! Thanks for elaborating.